config/file_test.go

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Stefan Prodan 2017. All rights reserved.
```

Comment:
Can you update this per - https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md#copyright-notice